guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -41,6 +41,46 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
+Configure the Discovery Service on the applicable {EL} versions below.
+
+* xref:#configuring-discovery-el-8[{EL} 8]
+* xref:#configuring-discovery-el-7[{EL} 7]
```

Comment:
There is no difference in instructions between EL 7 & 8: the installer takes care of all of that. Please align on only using the installer for both EL 7 & 8.

Reply:
Addressed :+1:

Reply:
Addressed :+1: 
Removing the separate section for EL7 as the commands from EL8 are applicable to both, as per my conversation with SME.